cmd/oc/main.go

```diff
@@ -29,16 +29,24 @@ import (
 	expect "github.com/ryandgoulding/goexpect"
 )
 
+const (
+	// incorrectUsageExitCode is the Unix return code used when the supplied arguments are inappropriate.
+	incorrectUsageExitCode = 2
+
+	// mandatoryNumArgs is the number of positional arguments required
```

Comment:
Please end with a period to be consistent with the comment for `incorrectUsageExitCode`.